### PR TITLE
Pipeline changes

### PR DESCRIPTION
**Pull request by tarensanders**
Makes a few changes that should make the pipeline run a bit better on the HPC. See #72.

I also added GCP storage for data_imp. It *should* fit well within the free tier limit, but if it doesn't we can easily remove it.

To setup your own access, follow the setup steps in the [targets manual](https://books.ropensci.org/targets/data.html#gcp-setup). But, you'll need to configure the oAuth JSON to use the bucket I've shared with you. hopefully I've given you the right permissions.

Note that for this to work/actually save us time we need to push the `_targets/meta/meta` file to github so that we are sharing an object hash.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,9 +35,6 @@ vignettes/*.pdf
 # Shiny token, see https://shiny.rstudio.com/articles/shinyapps.html
 rsconnect/
 
-# targets
-_targets/
```
Comment (conig) [inline .gitignore:39]: Wouldn't this mean targets objects get uploaded to github including data files?

**Review by conig — APPROVED** [review]
looks good, just check you're happy that targets objects including data won't get uploaded to the github repo with the changes to git ignore.

File: .gitignore (hunk 1 above)
Comment (tarensanders) [reply .gitignore:39]: There's another .gitignore in the _targets folder that excludes everything except itself and the meta file. That comes baked in with the default targets setup, for exactly this reason.

**Comment by tarensanders** [comment]
Before I merge this I have one quick (I hope) additional suggestion.

I was going to also put all the `model_list_*` on GCP, but a quick check shows that they are >500mb each (and even larger in memory)

```r
r$> library(lobstr) 
library(butcher) 
     
    targets::tar_load(model_list_by_weekday) 
     
    obj_size(model_list_by_weekday) 
1.57 GB
```

Each of the models from the imps is ~100mb, so that's 300mb for 3 imps per exposure/outcome.

```r
r$> obj_size(model_list_by_weekday$`scale_pa_intensity by scale_sleep_regularity_lag`$model$`3`)
99.99 MB
```

For whatever reason, this is all because of the size of the `@call$formula` environment. 

```r
r$> obj_size(model_list_by_weekday$`scale_pa_intensity by scale_sleep_regularity_lag`$model$`3`@call$formula)
98.76 MB
```

If you remove this, it comes down to like 3kb.

```r
r$> model_list_by_weekday$`scale_pa_intensity by scale_sleep_regularity_lag`$model$`3`@call$formula <- butcher::axe_env(model_list_by_weekday$`scale_pa_intensity by scale_sleep_regularity_lag`$model$`3`@call$formula) 
     
    obj_size(model_list_by_weekday$`scale_pa_intensity by scale_sleep_regularity_lag`$model$`3`@call$formula)                                                                                                                    
3.78 kB
```

So, I'm going to remove this environment before the model is returned, and check if the pipeline still runs. I'll do that before I merge this.

**Pull request merged**